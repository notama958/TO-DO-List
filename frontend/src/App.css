/* Global Styles */
:root {
  --primary-color: #17a2b8;
  --dark-color: #343a40;
  --light-color: #f4f4f4;
  --danger-color: #dc3545;
  --success-color: #28a745;
}

* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}

body {
  font-family: 'Raleway', sans-serif;
  font-size: 1rem;
  line-height: 1.6;
  color: #fff;
}

a {
  color: var(--primary-color);
  text-decoration: none;
}

ul {
  list-style: none;
}

img {
  width: 100%;
}

/* Utilities */
.container {
  max-width: 1100px;
  max-height: 100%;
  margin: auto;
  overflow: hidden;
  padding: 0 2rem;
  margin-top: 6rem;
  margin-bottom: 3rem;
  position: relative;
}

/* Text Styles*/
.x-large {
  font-size: 4rem;
  line-height: 1.2;
  margin-bottom: 1rem;
}

.large {
  font-size: 3rem;
  line-height: 1.2;
  margin-bottom: 1rem;
}

.lead {
  font-size: 1.5rem;
  margin-bottom: 1rem;
}

.text-center {
  text-align: center;
}

.text-primary {
  color: var(--primary-color);
}

.text-dark {
  color: var(--dark-color);
}

/* Padding */
.p {
  padding: 0.5rem;
}
.p-1 {
  padding: 1rem;
}
.p-2 {
  padding: 2rem;
}
.p-3 {
  padding: 3rem;
}
.py {
  padding: 0.5rem 0;
}
.py-1 {
  padding: 1rem 0;
}
.py-2 {
  padding: 2rem 0;
}
.py-3 {
  padding: 3rem 0;
}

/* Margin */
.m {
  margin: 0.5rem;
}
.m-1 {
  margin: 1rem;
}
.m-2 {
  margin: 2rem;
}
.m-3 {
  margin: 3rem;
}
.my {
  margin: 0.5rem 0;
}
.my-1 {
  margin: 1rem 0;
}
.my-2 {
  margin: 2rem 0;
}
.my-3 {
  margin: 3rem 0;
}

.btn {
  display: inline-block;
  background: var(--light-color);
  color: #333;
  padding: 0.4rem 1.3rem;
  font-size: 1rem;
  border: none;
  cursor: pointer;
  margin-right: 0.5rem;
  transition: opacity 0.2s ease-in;
  outline: none;
}

.badge {
  font-size: 0.8rem;
  padding: 0.1rem;
  text-align: center;
  margin: 0.3rem;
  background: var(--light-color);
  color: #333;
}

.alert {
  padding: 0.8rem;
  margin: 1rem 0;
  opacity: 0.9;
  background: var(--light-color);
  color: #333;
}

.btn-primary,
.bg-primary,
.badge-primary,
.alert-primary {
  background: var(--primary-color);
  color: #fff;
}

.btn-light,
.bg-light,
.badge-light,
.alert-light {
  background: var(--light-color);
  color: #333;
}

.btn-dark,
.bg-dark,
.badge-dark,
.alert-dark {
  background: var(--dark-color);
  color: rgb(192, 186, 186);
}

.btn-danger,
.bg-danger,
.badge-danger,
.alert-danger {
  background: var(--danger-color);
  color: #fff;
}

.btn-success,
.bg-success,
.badge-success,
.alert-success {
  background: var(--success-color);
  color: #fff;
}

.btn-white,
.bg-white,
.badge-white,
.alert-white {
  background: #fff;
  color: #333;
  border: #ccc solid 1px;
}

.btn:hover {
  opacity: 0.8;
}

.bg-light,
.badge-light {
  border: #ccc solid 1px;
}

.round-img {
  border-radius: 50%;
}

.line {
  height: 1px;
  background: #ccc;
  margin: 1.5rem 0;
}

/* Overlay */
.dark-overlay {
  background-color: rgba(0, 0, 0, 0.7);
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
}

/* Forms */
.form .form-group {
  margin: 1.2rem 0;
}

.form .form-text {
  display: block;
  margin-top: 0.3rem;
  color: #888;
}

.form input[type='text'],
.form input[type='email'],
.form input[type='password'],
.form input[type='date'],
.form input[type='number'],
.form select,
.form textarea {
  display: block;
  width: 100%;
  padding: 0.4rem;
  font-size: 1.2rem;
  border: 1px solid #ccc;
}

.form input[type='submit'],
button {
  font: inherit;
}
.form input[type='cancel'],
button {
  font: inherit;
}

.form .social-input {
  display: flex;
}

.form .social-input i {
  padding: 0.5rem;
  width: 4rem;
}

.form .social-input i.fa-twitter {
  color: #38a1f3;
}
.form .social-input i.fa-facebook {
  color: #3b5998;
}
.form .social-input i.fa-instagram {
  color: #3f729b;
}
.form .social-input i.fa-youtube {
  color: #c4302b;
}
.form .social-input i.fa-linkedin {
  color: #0077b5;
}

.table th,
.table td {
  padding: 1rem;
  text-align: left;
}

.table th {
  background: var(--light-color);
}

/* Navbar */
.navbar {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0.7rem 2rem;
  position: fixed;
  z-index: 1;
  width: 100%;
  top: 0;
  border-bottom: solid 1px var(--primary-color);
  opacity: 0.9;
  background: whitesmoke;
}

.navbar ul {
  display: flex;
}

.navbar a {
  color: #333;
  padding: 0.45rem;
  margin: 0 0.25rem;
}

.navbar a:hover {
  color: var(--primary-color);
}

.navbar .welcome span {
  margin-right: 0.6rem;
}

/* Landing Page */
.landing {
  position: relative;
  /* background: url('../img/showcase.jpg') no-repeat center center/cover; */
  height: 100vh;
}

.landing-inner {
  color: #fff;
  height: 100%;
  width: 80%;
  margin: auto;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  text-align: center;
}

/* Profiles Page */
.profile {
  display: grid;
  grid-template-columns: 2fr 2fr 4fr;
  align-items: center;
  grid-gap: 2rem;
  padding: 1rem;
  line-height: 1.8;
  margin-bottom: 1rem;
}

/* Posts Page */
.post-form .post-form-header {
  background: var(--primary-color);
  padding: 0.5rem;
}

.post {
  display: grid;
  grid-template-columns: 1fr 4fr;
  grid-gap: 2rem;
  align-items: center;
}

.post > div:first-child {
  text-align: center;
}

.post img {
  width: 100px;
}

.post .comment-count {
  background: var(--light-color);
  color: var(--primary-color);
  padding: 0.1rem 0.2rem;
  border-radius: 5px;
  font-size: 0.8rem;
}

.post .post-date {
  color: #aaa;
  font-size: 0.8rem;
  margin-bottom: 0.5rem;
}

/* Mobile Styles */
@media (max-width: 700px) {
  .container {
    margin-top: 8rem;
  }

  .hide-sm {
    display: none;
  }

  /* Text Styles */
  .x-large {
    font-size: 3rem;
  }

  .large {
    font-size: 2rem;
  }

  .lead {
    font-size: 1rem;
  }

  /* Navbar */
  .navbar {
    display: block;
    text-align: center;
  }

  .navbar ul {
    text-align: center;
    justify-content: center;
  }

  .navbar h1 {
    margin-bottom: 1rem;
  }

  .navbar .welcome {
    display: none;
  }

  /* Profiles Page */
  .profile {
    grid-template-columns: 1fr;
    text-align: center;
  }

  .profile ul {
    display: none;
  }
}

button:focus {
  outline: none;
}
.cmds {
  display: block;
  position: relative;
  margin: auto;
  text-align: center;
}
#title {
  display: block;
  position: relative;
  margin: auto;
}
#title h1 {
  margin: 0;
  color: black;
  font-size: 1.5rem;
  text-align: left;
}

#title button {
  font: inherit;
  padding: 0.5rem 1rem;
  background: #f67722;
  border: 1px solid #f67722;
  color: white;
  border-radius: 6px;
  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.26);
  cursor: pointer;
  margin-top: 15px;
}

#title button:hover,
#title button:active {
  background: #f3cc4b;
  border-color: #f3cc4b;
  color: #995200;
}
#title .btn--green {
  font: inherit;
  padding: 0.5rem 1rem;
  background: green;
  border: 1px solid #22f650;
  color: white;
  border-radius: 6px;
  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.26);
  cursor: pointer;
  margin-top: 15px;
}

#title .btn--green:hover,
#title .btn--green:active {
  background: greenyellow;
  border-color: greenyellow;
  color: #995200;
}

label {
  font-weight: bold;
  margin: 0.5rem 0;
  color: #464646;
}

input {
  font: inherit;
  border: 1px solid #ccc;
  padding: 0.4rem 0.2rem;
  color: #383838;
}

input:focus {
  outline: none;
  background: #fff1c4;
}

.btn:hover,
.btn:active {
  background: #f67722;
  border-color: #f67722;
}

.btn--passive {
  color: #00329e;
  background: transparent;
  border: none;
  box-shadow: none;
}

.btn--passive:hover,
.btn--passive:active {
  background: #aec6f8;
}

.btn--danger {
  background: #d30808;
  color: white;
  border-color: #d30808;
}

.btn--danger:hover,
.btn--danger:active {
  background: #ff3217;
  border-color: #ff3217;
}

.card {
  background: white;
  border-radius: 10px;
  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.26);
}
.card li {
  position: relative;
  margin-top: 11px;
}
.modal,
#add-modal {
  position: inherit;
  z-index: 1000;
  width: 80%;
  top: 5vh;
  left: 10%;
  /* display: none; */
  display: block;
  animation: fade-slide-in 0.3s ease-out forwards;
}

.modal.visible {
  display: block;
  animation: fade-slide-in 0.3s ease-out forwards;
}

.modal .modal__title {
  margin: 0;
  padding: 1rem;
  border-bottom: 1px solid #00329e;
  background: #00329e;
  color: white;
  border-radius: 10px 10px 0 0;
}

.modal .modal__content {
  padding: 1rem;
  color: var(--dark-color);
}

.modal .modal__actions {
  padding: 1rem;
  display: flex;
  justify-content: flex-end;
}

#id-text {
  width: 40rem;
  margin: 2rem auto;
  max-width: 80%;
  padding: 1rem;
  color: black;
}
#id-text p {
  font-size: 1.5rem;
  text-align: center;
}
#entry-text {
  width: 40rem;
  margin: 2rem auto;
  max-width: 80%;
  padding: 1rem;
  color: black;
}

#entry-text p {
  font-size: 1.5rem;
  text-align: center;
}

#backdrop {
  position: fixed;
  width: 100%;
  height: 100vh;
  top: 0;
  left: 0;
  background: rgba(0, 0, 0, 0.75);
  z-index: 10;
  pointer-events: none;
  display: block;

  pointer-events: all;
  /* display: none; */
}

#backdrop.visible {
  display: block;
  pointer-events: all;
}

#add-modal .modal__content {
  display: flex;
  flex-direction: column;
  z-index: 1000;
}

@media (max-width: 768px) {
  .modal {
    max-width: 100%;
  }
  #list-id {
    max-width: 100%;
    left: calc(50% - 20rem);
  }
  #list-content {
    max-width: 100%;
    left: calc(50% - 10rem);
  }
}

@keyframes fade-slide-in {
  from {
    transform: translateY(-5rem);
  }
  to {
    transform: translateY(0);
  }
}
.tableList {
  border: 1px solid black;
  width: 100%;
  display: table;
  position: relative;
}
#taskValue {
  margin: 0;
  border-radius: 0;
  width: 75%;
  padding: 10px;
  float: left;
  font-size: 16px;
}

#createListNode {
  position: absolute;
  padding: 10px;
  width: 25%;
  float: right;
  text-align: center;
  font-size: 16px;
  cursor: pointer;
  border-radius: 0;
  background: #d9d9d9;
}

/* Remove margins and padding from the list */
#task-list {
  margin: 0;
  padding: 0;
  position: relative;
}
.task-element {
  font-weight: bold;
  margin: 0.5rem 0;
  color: #464646;
  width: 100%;
  position: relative;
}

/* Style the list items */
#task-list li {
  border: 1px solid black;
  cursor: pointer;
  position: inherit;
  padding: 12px 8px 12px 40px;
  list-style-type: none;
  font-size: 18px;
  transition: 0.2s;

  /* make the list items unselectable */
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

/* Darker background-color on hover */
.task-element:hover {
  background: #ddd;
}

/* When clicked on, add a background color and strike out text */
#task-list li.checked {
  background: #888;
  color: #fff;
  text-decoration: line-through;
  text-decoration-color: black;
}

/* Add a "checked" mark when clicked on */
#task-list li.checked::before {
  content: '';
  position: absolute;
  border-color: #fff;
  border-style: solid;
  border-width: 0 2px 2px 0;
  top: 10px;
  left: 16px;
  transform: rotate(45deg);
  height: 15px;
  width: 7px;
}
.close {
  position: absolute;
  top: 0;
  bottom: 0;
  right: 0;
  margin: auto;
  display: inline-flex;
  align-items: center;
  z-index: 1000;
}

/*
*
*
*
* 
edit dashboard
*
*
*
*
*/
#list-id {
  list-style: none;
  width: 50rem;
  margin: 1rem auto;
  padding: 0;
  color: black;
  text-align: center;
  font-size: 1.2rem;
}
#list-id li {
  display: block;
}
#list-id p {
  background: var(--light-color);
  color: var(--primary-color);
  padding: 0.1rem 0.2rem;
  border-radius: 5px;
  font-weight: bold;
}
#list-content {
  list-style: none;
  width: 50rem;
  margin: 0.5rem auto;
  margin-top: 10px;
  margin-bottom: 0;
  padding: 0;
  flex-direction: column;
  height: 100%;
}
.list-element {
  margin: 1rem;
  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.26);
  border-radius: 10px;
  display: flex;
  position: inherit;
}

.list-element h2,
.list-element p {
  font-size: 1.25rem;
  margin: 0;
}

.list-element h2 {
  color: #f4f4f4;
  margin-bottom: 1rem;
}

.list-element p {
  color: white;
  display: inline;
  background: #f67722;
  padding: 0.25rem 1rem;
  border-radius: 15px;
}

.list-element__image {
  flex: 1;
  width: 10rem;
  border-radius: 20px 20px 20px 20px;
  position: relative;
}

.list-element__image img {
  object-fit: cover;
  position: absolute;
  align-items: center;
  margin: auto;
  vertical-align: middle;
  top: 0;
  bottom: 0;
}

.list-element__info {
  flex: 3;
  padding: 5px;
  position: relative;
  width: 10rem;
}
.wrapper-title {
  margin: 2px;
  padding: 1rem;
  display: block;
  position: inherit;
  text-align: center;
  word-wrap: break-word;
}
.wrapper-content {
  margin: 2px;
  padding: 1rem;
  display: block;
  position: inherit;
  text-align: center;
}

/* Style the list items */
.wrapper-content ul {
  position: inherit;
}
.wrapper-content li {
  border: 1px solid black;
  cursor: pointer;
  position: inherit;
  list-style-type: none;
  font-size: 18px;
  transition: 0.2s;
  text-align: center;
  /* make the list items unselectable */
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

/* Darker background-color on hover */
.task-element:hover {
  background: #ddd;
}

/* When clicked on, add a background color and strike out text */
.wrapper-content li.checked {
  background: #888;
  color: #fff;
  text-decoration: line-through;
  text-decoration-color: black;
}

/* Add a "checked" mark when clicked on */
.wrapper-content li.checked::before {
  content: '';
  position: absolute;
  border-color: #fff;
  border-style: solid;
  border-width: 0 2px 2px 0;
  top: auto;
  bottom: auto;
  left: 16px;
  transform: rotate(45deg);
  height: 15px;
  width: 7px;
  border: 1;
}
.wrapper-content li .close {
  position: absolute;
  top: 0;
  bottom: 0;
  right: 0;
  margin: auto;
  display: inline-flex;
  align-items: center;
}

.delete,
.update {
  position: relative;
  align-items: center;
  margin: 0.5rem;
}
