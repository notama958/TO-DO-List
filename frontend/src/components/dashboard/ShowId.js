import React, { Fragment } from 'react';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';

const ShowId = ({ listid }) => {
  return (
    <Fragment>
      <li className=" fas fa-check-circle  ">{listid}</li>
    </Fragment>
  );
};

ShowId.propTypes = {
  listid: PropTypes.string.isRequired,
};

export default ShowId;
