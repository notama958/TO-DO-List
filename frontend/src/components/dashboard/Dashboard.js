import React, { Fragment, useState, useEffect, useRef } from 'react';
import PropTypes from 'prop-types';
import {
  modalToggle,
  backdropToggle,
  getRandomList,
  getListById,
  addRandom,
} from '../../actions/list';
import ListForm from './ListForm';
import Backdrop from '../layout/Backdrop';
import ShowContent from './ShowContent';
import ShowId from './ShowId';
import UpdateList from './UpdateList';
import Spinner from '../layout/Spinner';
import { connect } from 'react-redux';
import { Provider } from 'react-redux';
import store from '../../store';
const Dashboard = ({
  list: { modal, backdrop, loading, ids, lists, edit },
  modalToggle,
  backdropToggle,
  getRandomList,
  getListById,
  addRandom,
}) => {
  const ID = useRef(null);
  return (
    <Provider store={store}>
      <header id="title" className="cmds">
        <label className="large text-primary post-form-header">NOTE ME</label>
        <hr />
        <button
          value="add"
          className="btn btn--green"
          onClick={() => {
            modalToggle(true);
            backdropToggle(true);
          }}
        >
          ADD{' '}
        </button>
        <button
          value="add"
          className="btn btn--green"
          onClick={() => {
            addRandom();
          }}
        >
          ADD RANDOM{' '}
        </button>
        <br />
        <button
          value="load"
          className="btn btn--danger"
          onClick={() => {
            getRandomList();
          }}
        >
          {' '}
          GET RANDOM{' '}
        </button>
        <div>
          <button
            value="get"
            className="btn btn--danger"
            onClick={() => {
              getListById(ID.current.value);
            }}
          >
            GET BY ID{' '}
          </button>
          <input type="text" name="id" ref={ID}></input>
        </div>
      </header>
      {backdrop === true ? <Backdrop /> : ' '}
      {modal === true ? <ListForm /> : ' '}
      {edit === true ? <UpdateList /> : ' '}
      {/*show items here */}
      <main>
        {ids.length <= 0 ? (
          <section id="id-text" className="card">
            <p>LIST ID shows here!</p>
          </section>
        ) : (
          <ul id="list-id" className="card">
            <p>Newly Created List ID</p>
            {ids.map((id, index) => (
              <ShowId key={index} listid={id} />
            ))}
          </ul>
        )}
        {lists.length <= 0 ? (
          <section id="entry-text" className="card">
            <p>List CONTENT shows here!</p>
          </section>
        ) : (
          <ul id="list-content">
            {lists.map((lst, index) => (
              <ShowContent
                key={index}
                mylist={lst}
                modal={modal}
                backdrop={backdrop}
              />
            ))}
          </ul>
        )}
      </main>
    </Provider>
  );
};

Dashboard.propTypes = {
  modalToggle: PropTypes.func.isRequired,
  backdropToggle: PropTypes.func.isRequired,
  getRandomList: PropTypes.func.isRequired,
  getListById: PropTypes.func.isRequired,
  addRandom: PropTypes.func.isRequired,
};
const mapStateToProps = (state) => ({
  list: state.list,
});

export default connect(mapStateToProps, {
  modalToggle,
  backdropToggle,
  getRandomList,
  getListById,
  addRandom,
})(Dashboard);
