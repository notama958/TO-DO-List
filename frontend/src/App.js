import './App.css';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
// react redux
import { Provider } from 'react-redux';
import store from './store';
import { Fragment } from 'react';
// components
import Landing from './components/layout/Landing';
import Alert from './components/layout/Alert';
import Navbar from './components/layout/Navbar';
import Dashboard from './components/dashboard/Dashboard';

const App = () => {
  return (
    <Provider store={store}>
      <Router>
        <Fragment>
          <Navbar />
          <Route exact path="/" component={Landing} />
          <section className="container">
            <Alert />
            <Switch>
              <Route exact path="/note" component={Dashboard} />
            </Switch>
          </section>
        </Fragment>
      </Router>
    </Provider>
  );
};

export default App;
