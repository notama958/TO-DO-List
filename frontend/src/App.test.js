import { render, screen, act } from '@testing-library/react';
import App from './App';
import Dashboard from '../src/components/dashboard/Dashboard';
import ShowId from '../src/components/dashboard/ShowId';
import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
describe('Render cover page', () => {
  it('Checking title', () => {
    act(() => {
      render(<App />);
    });
    const linkElement = screen.getByText(/Take a note/i);
    expect(linkElement).toBeInTheDocument();
  });
});
const mockStore = configureStore([]);

describe('Redux component tests', () => {
  let store;
  let component, showIds;
  beforeEach(() => {
    store = mockStore({
      list: {
        list: null,
        ids: [], // list of id
        lists: [], // get one list full info
        loading: true,
        error: {},
        modal: false,
        edit: false,
        backdrop: true,
        tasks: [],
      },
    });
    store.dispatch = jest.fn();
    component = renderer.create(
      <Provider store={store}>
        <Dashboard />
      </Provider>
    );
    const id = '12121212';
    showIds = renderer.create(
      <Provider store={store}>
        <ShowId listid={id} />
      </Provider>
    );
  });
  it('Render Dashboard component', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
  it('Render get random button', () => {
    renderer.act(() => {
      component.root.findAllByType('button')[1].props.onClick();
    });
    expect(store.dispatch).toHaveBeenCalledTimes(1);
    expect(store.dispatch).toHaveBeenCalled(); // anomynous func
  });
  it('Render show Id', () => {
    expect(showIds.toJSON()).toMatchSnapshot();
  });
});
