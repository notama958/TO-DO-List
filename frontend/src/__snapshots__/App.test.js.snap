// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Redux component tests Render Dashboard component 1`] = `
Array [
  <header
    className="cmds"
    id="title"
  >
    <label
      className="large text-primary post-form-header"
    >
      NOTE ME
    </label>
    <hr />
    <button
      className="btn btn--green"
      onClick={[Function]}
      value="add"
    >
      ADD
       
    </button>
    <button
      className="btn btn--green"
      onClick={[Function]}
      value="add"
    >
      ADD RANDOM
       
    </button>
    <br />
    <button
      className="btn btn--danger"
      onClick={[Function]}
      value="load"
    >
       
      GET RANDOM
       
    </button>
    <div>
      <button
        className="btn btn--danger"
        onClick={[Function]}
        value="get"
      >
        GET BY ID
         
      </button>
      <input
        name="id"
        type="text"
      />
    </div>
  </header>,
  <div
    id="backdrop"
  />,
  " ",
  " ",
  <main>
    <section
      className="card"
      id="id-text"
    >
      <p>
        LIST ID shows here!
      </p>
    </section>
    <section
      className="card"
      id="entry-text"
    >
      <p>
        List CONTENT shows here!
      </p>
    </section>
  </main>,
]
`;

exports[`Redux component tests Render show Id 1`] = `
<li
  className=" fas fa-check-circle  "
>
  12121212
</li>
`;
